docs/kyma/docs/04-12-local-installation-scripts.md

```diff
@@ -3,7 +3,7 @@ title: Local installation scripts deep-dive
 type: Installation
 ---
 
-This document extends the [Install Kyma locally from sources](#installation-install-kyma-locally-from-sources) guide with a detailed breakdown of the alternative local installation method which is the `run.sh` script.
+This document extends the [Install Kyma locally](#installation-install-kyma-locally) guide with a detailed breakdown of the alternative local installation method which is the `run.sh` script.
```

Comment:
I guess we need to restructure this sentence as the `run.sh` is the main local installation method and not the alternative one, yup?